### DISCUSSION TITLE
Daily bills

### DISCUSSION THREAD

**Discussion by wi6436** [firefly-iii/firefly-iii#5690] [Firefly III]
The minimum repeat interval of bills, which I can choose, is 'weekly'. Is it somehow possible to enable a daily repeat interval?

Many thanks!

**Comment by JC5** [comment]
No. What kind of bill hits daily?

**Reply by wi6436** [reply]
Hello James,

thanks a lot for replying so quickly! It's a mobile contract, which can be booked via an app on a daily base (https://www.freenet-funk.de/). They also debit every single day, including on weekends.

**Reply by JC5** [reply]
Wow. That's annoying 😉 

I'll put it on the list!

**Reply by wi6436** [reply]
That is great, thanks! :thumbsup:
